heartbeat/monitors/active/http/http_test.go

```diff
@@ -257,12 +257,12 @@ func TestUpStatuses(t *testing.T) {
 
 				testslike.Test(
 					t,
-					lookslike.Strict(lookslike.Compose(
+					lookslike.Compose(
```

Comment:
Porting this change from main: https://github.com/elastic/beats/blob/5e71fe7a7947b844013218a8fe02dce04a142bd9/heartbeat/monitors/active/http/http_test.go#L262